From 9ac0967b4b3eb7c3e85534ad9cc2d8ea6f567443 Mon Sep 17 00:00:00 2001
From: Tao Ren <taoren@meta.com>
Date: Fri, 2 Feb 2024 16:55:06 -0800
Subject: [PATCH 1/8] ARM: dts: aspeed: Remove eMMC from
 ast2600-facebook-netbmc-common.dtsi

Remove eMMC device entries from ast2600-facebook-netbmc-common.dtsi
because eMMC will be removed from future Meta/Facebook AST2600 network
OpenBMC platforms.
---
 .../dts/aspeed/ast2600-facebook-netbmc-common.dtsi   | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/ast2600-facebook-netbmc-common.dtsi b/arch/arm/boot/dts/aspeed/ast2600-facebook-netbmc-common.dtsi
index 00e5887c926f..83c9789d45b1 100644
--- a/arch/arm/boot/dts/aspeed/ast2600-facebook-netbmc-common.dtsi
+++ b/arch/arm/boot/dts/aspeed/ast2600-facebook-netbmc-common.dtsi
@@ -152,18 +152,6 @@ &vhub {
 	status = "okay";
 };
 
-&emmc_controller {
-	status = "okay";
-};
-
-&emmc {
-	status = "okay";
-
-	non-removable;
-	max-frequency = <25000000>;
-	bus-width = <4>;
-};
-
 &rtc {
 	status = "okay";
 };
-- 
2.43.0

